server/routes/v1/upload.ts

```diff
@@ -40,7 +40,8 @@ export const postUpload = [
     const session = await mongoose.startSession()
     return session.withTransaction(async () => {
       const files = req.files as unknown as MinioFile
-      const mode = req.query.mode
+      const mode: string = req.query.mode as string
```

Comment:
I don't think you need `: string` here, as TypeScript should already know it's a string.

Reply:
Resvoled